### ISSUE TITLE
Missing Aria-labels on the text document icons present on the Sample Queries 

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#2235]
**Expected behavior**
Links should have an Aria-label present

**Screenshots**
![image](https://user-images.githubusercontent.com/18407044/201632872-5d5b8e6f-47f1-421c-8eb6-d1def1f3e0bf.png)

![image](https://user-images.githubusercontent.com/18407044/201632555-eaabd25f-bda0-4da6-b9c1-f867432808aa.png)
